scipy/special/_support_alternative_backends.py

```diff
@@ -51,22 +55,38 @@ def wrapped(*args, **kwargs):
 
 
 array_special_func_map = {
+    'logit': 1,
+    'expit': 1,
     'log_ndtr': 1,
     'ndtr': 1,
     'ndtri': 1,
+    'digamma': 1,
+    'polygamma': 2,
+    'multigammaln': 2,
+    'gammaln': 1,
+    'gamma': 1,
+    'gammainc': 2,
+    'gammaincc': 2,
+    'betaln': 2,
+    'beta': 2,
+    'betainc': 3,
     'erf': 1,
     'erfc': 1,
+    'erfinv': 1,
+    'zeta': 2,
+    # No one implements binom yet.
+    # 'binom': 2,
+    'expi': 1,
+    'expn': 2,
+    'softmax': 1,
```

Comment:
This function is not in `_ufuncs.py`. Even if you were to fix the import, it is not elementwise and cannot use this delegation system without structural changes to it. 

This delegation system can't realise that the function is already API agnostic, so it will try to convert everything to numpy before pushing data through it. On Dask, this will cause reductions to be applied locally to chunks instead of globally. On MArray, it will incorrectly calculate the mask.

I expect (hope) that the Dask test will fail  really loudly saying that the numbers coming out of it are wrong. Note that there is no test for MArray.

Reply:
Please don't overload this module, which was never meant for anything that is not an elementwise ufunc and is not fit for purpose for it.
I'm very happy to go through a more structural redesign and unification of the delegation system, but it should be its own separate design effort and PR. This is out of scope here.

Reply:
Removed `softmax`, and `log_softmax`